### PR TITLE
[zh-cn]: update the translation of css `url()` function

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/url
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/url/index.md
* Last commit: https://github.com/mdn/content/commit/875d97804293021faaf66c16a76e2f31a6df56e1

### CODE REVIEW

File: files/zh-cn/web/css/url/index.md [hunk 1]
```diff
@@ -1,60 +1,187 @@
...
-[CSS](/zh-CN/CSS) 数据类型 `<url>` 指向一个资源。它没有独有的表达形式，只能通过 `url()` 函数定义。
-
-> **备注：** **URI 和 URL?**
->
-> [URI](https://zh.wikipedia.org/wiki/%E7%BB%9F%E4%B8%80%E8%B5%84%E6%BA%90%E6%A0%87%E5%BF%97%E7%AC%A6)（统一资源标识符）与 [URL](https://zh.wikipedia.org/wiki/%E7%BB%9F%E4%B8%80%E8%B5%84%E6%BA%90%E5%AE%9A%E4%BD%8D%E7%AC%A6)（统一资源定位符）不同。URL 描述资源的位置，而 URI 描述资源的 id。URI 可以是一个资源的 URL（地址）、或 [URN](https://zh.wikipedia.org/wiki/%E7%BB%9F%E4%B8%80%E8%B5%84%E6%BA%90%E5%90%8D%E7%A7%B0)（统一资源名称）。
->
-> 在 CSS Level 1 中，`url()` 函数被引入并用于描述 URL，即地址（虽然没有明确定义，但指一个 CSS 数据类型 `<url>`）
->
-> 在 CSS Level 2 中，`url()` 函数被扩展为可以描述任何一个 URI，即 URL 或 URN。这一定义导致 `url()` 函数被用于创建一个 `<uri>` 数据类型。这一行为使人迷惑，且在 CSS 中几乎从不使用 URN。
+> **备注：** {{Glossary("URI", "统一资源标识符")}}和{{Glossary("URL", "统一资源定位器")}}是不同的。URI 用于标识资源。URL 是 URI 的一种类型，用于描述资源的*位置*。URI 可以是 URL，也可以是资源的名称（{{Glossary("URN", "统一资源名称")}}）。
```
Comment (T34-active) [inline files/zh-cn/web/css/url/index.md:12]:
```suggestion
> **备注：** {{Glossary("URI")}} 和 {{Glossary("URL")}} 是不同的。URI 用于标识资源。URL 是 URI 的一种类型，用于描述资源的*位置*。URI 可以是 URL，也可以是 {{Glossary("URN")}}（资源的名称）。
```

**Review by jasonren0403 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/css/url/index.md [hunk 2]
```diff
@@ -1,60 +1,187 @@
...
+```
+
+{{EmbedLiveSample("使用数据 URL", "100%", 50)}}
+
+### 在 filters 中的使用
+
+当 URL 被用作 filter 的路径时，URL 必须是以下其中之一：
+
+1. SVG 文件的路径，并附加 filters 的 ID。
+2. 如果页面上已经存在 SVG，则为 filters 的 ID。
```
Comment (T34-active) [inline files/zh-cn/web/css/url/index.md:161]:
```suggestion
当 URL 被用作滤镜路径时，URL 必须是以下其中之一：

1. SVG 文件的路径，并附加滤镜的 ID。
2. 如果页面上已经存在 SVG，则为滤镜的 ID。
```

File: files/zh-cn/web/css/url/index.md [hunk 3]
```diff
@@ -1,60 +1,187 @@
...
+
+```css
+.blur {
+  filter: url(my-file.svg#svg-blur); /* 作为 filter 使用的 SVG 文件的 URL */
```
Comment (T34-active) [inline files/zh-cn/web/css/url/index.md:165]:
```suggestion
  filter: url(my-file.svg#svg-blur); /* 作为滤镜使用的 SVG 文件的 URL */
```

**Review by T34-active — COMMENTED** [review]
fix filters

File: files/zh-cn/web/css/url/index.md [hunk 4]
```diff
@@ -1,60 +1,187 @@
...
+        ```
+
+        如果选择不使用引号编写 URL，请在 URL 中的任何括号、空白字符、单引号（`'`）和双引号（`"`）前使用反斜线（`\`）。
 
-> **备注：** 从 Firefox 15 开始，不再允许在未用引号包含的 url() 中使用大于 0x7e 的控制字符。详情请查看 [Firefox bug 752230](https://bugzil.la/752230)。
+    - path
+      - : 引用 [SVG 形状](/zh-CN/docs/Web/SVG/Tutorial/Basic_Shapes)（`circle`、`ellipse`、`line`、`path`、`polygon`、`polyline` 或 `rect`）的 ID ——使用形状的几何体作为路径。
+
+- `<url-modifier>`
+  - 将来，`url()` 函数可能会支持指定修改器、标识符或函数标记，从而改变 URL 字符串的含义。本规范不支持此特性，也未对其进行全面定义。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/url/index.md:81]:
```suggestion
  - 将来，`url()` 函数可能会支持指定修饰符、标识符或函数标记，从而改变 URL 字符串的含义。本规范不支持此特性，也未对其进行全面定义。
```

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**